.github/issue_templates/Algorithim_development.md

```diff
@@ -0,0 +1,26 @@
+### Algorithm Description:
+
+### Requirements:
+
+> **Specify: (optional)**
+
+### Algorithm Code Info: Data/Input/Output
+
+> **Specify details:**
+
+### Code:
+
+```python
+# Code
+```
+
+### Requirements/ Dependencies:
+
+- [ ] 
+    1. Yes
+- [ ] 
+    2. No
```

Comment:
This section doesn't render quite right in Markdown: 
![image](https://github.com/IMAP-Science-Operations-Center/imap_processing/assets/12417828/93b43579-6f16-4ec7-b328-4dcb7083d783)

Using checkboxes will show "1/2" items completed, so I don't think we want them. Perhaps just remove these and put `<!-- Specify if any dependencies were added -->` as a comment line in here?